website/blog/2021-03-21-Partials-Support-In-Prefabs.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "Partials Support in Prefabs"
+author: Nikhilesh K V
+---
+
+[Prefabs](/learn/app-development/custom-widgets/prefabs-overview) are reusable application parts that can be easily integrated into your applications. To learn how Prefabs can be created in WaveMaker, refer [this link](/learn/app-development/custom-widgets/creating-prefabs).
+
+<!--truncate-->
+
+## Understanding the Problem
+
+Until now, we did not allow users to create [partials](/learn/app-development/ui-design/page-concepts/partial-pages) inside Prefabs. The only page you have access to is the Main Page where all code exists. Due to this, the following problems can be seen:
```

Comment:
Instead of problem - solution structure, lets explain usecases which this will make possible